src/app/_components/Card/CardSpeсialist/ExperienceList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import P from 'prop-types';
+import { cn } from '@/utils/cn';
+import { ExperienceListItem } from './ExperienceListItem';
+import { specialistLabelPropType } from '@/app/_components/Card/CardSpeсialist/prop-types';
+import { borderStyle } from '@/app/_components/Card/CardSpeсialist/config';
+
+export function ExperienceList({ labels, className }) {
```

Comment:
🟢 consider renaming this and other ExperienceList components more appropriately, for ex. BadgeList

Reply:
BadgeList seems better, fixed it.